### PR TITLE
Update AppConfiguration .NET Provider package for 7.1.0-preview

### PR DESCRIPTION
**Pull request by amerjusupovic**

### CODE REVIEW

File: _data/releases/latest/dotnet-packages.csv [hunk 1]
```diff
@@ -1,7 +1,7 @@
 "Package","VersionGA","VersionPreview","DisplayName","ServiceName","RepoPath","MSDocs","GHDocs","Type","New","PlannedVersions","LatestGADate","FirstGADate","Support","EOLDate","Hide","Replace","ReplaceGuide","MSDocService","ServiceId","Notes"
 "Azure.AI.AnomalyDetector","","3.0.0-preview.7","Anomaly Detector","Cognitive Services","anomalydetector","","","client","true","","","","","","","Microsoft.Azure.CognitiveServices.AnomalyDetector","","","",""
 "Azure.Data.AppConfiguration","1.3.0","","App Configuration","App Configuration","appconfiguration","","","client","true","","11/08/2023","01/07/2020","active","","","","","","",""
-"Microsoft.Extensions.Configuration.AzureAppConfiguration","7.0.0","","App Configuration Provider","App Configuration","https://github.com/Azure/AppConfiguration-DotnetProvider","NA","NA","client","true","","","","","","","","","","",""
+"Microsoft.Extensions.Configuration.AzureAppConfiguration","7.0.0","7.1.0-preview","App Configuration Provider","App Configuration","https://github.com/Azure/AppConfiguration-DotnetProvider","NA","NA","client","true","","","","","","","","","","",""
```
Comment (weshaggard) [inline _data/releases/latest/dotnet-packages.csv:4]: Why does this need to be manually updated? The automation would update these version numbers automatically if the package adds azure-sdk as an owner.

Comment (weshaggard) [reply _data/releases/latest/dotnet-packages.csv:4]: Do you know if this is for docs or for something else? 

cc @danieljurek

Comment (amerjusupovic) [reply _data/releases/latest/dotnet-packages.csv:4]: We were told it's for docs, along with the azure-docs-sdk-dotnet repo.

Comment (weshaggard) [reply _data/releases/latest/dotnet-packages.csv:4]: OK we can manually add this for now but we probably should consider adding the azure-sdk account as an owner on nuget.

**Review by weshaggard — APPROVED** [review]

File: _data/releases/latest/dotnet-packages.csv (hunk 1 above)
Comment (jimmyca15) [reply _data/releases/latest/dotnet-packages.csv:4]: @weshaggard I haven't heard of that workflow for automation. Any pointer to a doc for that to ensure we are setting things up as expected?

Comment (weshaggard) [reply _data/releases/latest/dotnet-packages.csv:4]: I don't believe there is any documentation for this but our automation automactically queries nuget for packages that the "azure-sdk" account owns and will submit version updates to this file once it detects new versions have been published. So if you want this package version to be updated automatically then you would want to add "azure-sdk" as an owner of the package.

Comment (jimmyca15) [reply _data/releases/latest/dotnet-packages.csv:4]: Ah, sounds easy enough. Thanks.

Comment (weshaggard) [reply _data/releases/latest/dotnet-packages.csv:4]: If you do grant ownership ping afterwards as I will have to accept the ownership request.

**Pull request merged**